### PR TITLE
feat(breadcrumb-core): Add component - TWIG-97

### PR DESCRIPTION
**Pull request by planctus**
# PR description

Please drop a few lines about the PR: what it does, how to test it, etc.

## QA Checklist

In order to ensure a safe and quick review, please check that your PR follow those guidelines:

- [ ] I have put the vanilla component as `devDependencies`
- [ ] I have put the specs package as `devDependencies`
- [ ] I have added the components directly used in the twig file (with `include` or `embed`) as `dependencies`
- [ ] My component is listed in `@ecl-twig/ec-components`'s `dependencies`
- [ ] My variables naming follow the guidelines (snake case for twig)
- [ ] I have provided tests
- [ ] I have provided documentation (for the "notes" tab)
- [ ] If my local `yarn.lock` contains changes, I have committed it
- [ ] I have given my PR the proper label (`pr: review needed` to indicate that I'm done and now waiting for a review ,`pr: wip` to indicate that I'm actively working on it ...)

### CODE REVIEW

File: src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+{% set _ellipsis_label = ellipsis_label|default('') %}
+{% set _ellipsis_text = ellipsis_text|default('…') %}
+{% set _breadcrumb_segment_class = 'ecl-breadcrumb-core__segment' %}
+{% set _min_items_left = min_items_left|default(1) %}
+{% set _min_items_right = min_items_right|default(2) %}
+{% set _nav_attributes = [
+  { name: 'aria-label', value: _navigation_text },
+  { name: 'data-ecl-breadcrumb-core', value: 'true' }]
+%}
+{% set _breadcrumb_icon = {
```
Comment (serotonine) [inline src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig]: What about if the user decide to change the icon ?

File: src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+{# Print the result #}
+
+<nav class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  <ol class="ecl-breadcrumb-core__container">
+    {% for key, link in links %}
+      {% set _breadcrumb_segment_attributes = 'data-ecl-breadcrumb-core-item="static"' %}
+      {# Last item is not a link. #}
+      {% if loop.last %}
+        {% set _breadcrumb_segment_attributes = _breadcrumb_segment_attributes ~ ' aria-current="page"' %}
+        <li class="{{ _breadcrumb_segment_class ~ ' ecl-breadcrumb-core__current-page' }}" {{ _breadcrumb_segment_attributes|raw }}>
```
Comment (serotonine) [inline src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig]: You could use _css_class {{ _css_class ~ '__current-page '}}

File: src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig (hunk 1 above)
Comment (planctus) [reply src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig]: That is not an option, @serotonine :)

File: src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig (hunk 2 above)
Comment (planctus) [reply src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig]: true but i find it very confusing, although we have sometimes used this approach, i would  avoid it and here i did that on purpose.

**Comment by planctus** [comment]
hey, @serotonine , thanks for the review, you asked for changes but i don't know exactly what to change, please read the comments i left and if you find them ok we might put this back in needs review for a second round by someone else.. ;)

File: src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+              {% include '@ecl-twig/ec-component-icon/icon.html.twig' with _breadcrumb_icon only %}
+            </li>
+          {% endif %}
+        {% endif %}
+        {# Render all the items with different attributes, aria labels are set via js #}
+        <li class="{{ _breadcrumb_segment_class }}" {{ _breadcrumb_segment_attributes|raw }}>
+          {% include '@ecl-twig/ec-component-link/link.html.twig' with {
+            link: {
+              type: 'standalone',
+              label: link.label,
```
[the remark this answers is not in the document]
Comment (planctus) [reply src/ec/packages/ec-component-breadcrumb-core/breadcrumb-core.html.twig]: true, @serotonine , thanks for noticing it! ;)

**Pull request merged**